docs/flows.md

```diff
@@ -0,0 +1,54 @@
+# Windmill Flows 
+Windmill **Flows** allow you to streamline complex processes and operations by chaining together custom scripts to build dynamic apps. Each **Flow** is composed of one or more step scripts that can accept inputs, execute a function and then produce an output which can be used by subsequent step scripts. Flows contain all of the benefits of Windmill Scripts, offering immediate functionality such as third party app integrations, SQL queries, Slack, custom scripts and more!
+
+- **Flow Inputs** arguments for each flow can be edited using the flow input wizard or by editing their json-schema. 
+
+- **Step Scripts** can be dynamically added to each flow by clicking the '+' button. Scripts can be automatically imported from [Windmil Hub](https://hub.windmill.dev/) selecting a Hub path or by using your own custom scripts created in Windmill by selecting a Script path. 
+
+- **Step Script Inputs** can be added to each step script via static arguments or dynamic JavaScript arguments.
```

Comment:
would reword a little: "Each step has inputs that correspond to the input spec of the selected script. Those inputs can be filled statically or dynamically" or something to that effect